dagrunner/utils/__init__.py

```diff
@@ -33,7 +33,7 @@ def process_path(fpath: str) -> str:
     return fpath
 
 
-def get_proc_mem_stat(pid=os.getpid()):
+def get_proc_mem_stat(pid=None):
```

Comment:
this ensures that the reference documentation no longer has the pid number encoded in it (changing the documentation each time you build it).